CHANGELOG.md

```diff
@@ -4,6 +4,105 @@
+
+The browser SDKs now require ES2018+ compatible browsers. New minimum browser versions:
+
+Chrome 63 Edge 79 Safari/iOS Safari 12 Firefox 58 Opera 50 Samsung Internet 8.2
+
+For more details, please see the
+[version support section in migration guide](https://github.com/getsentry/sentry-javascript/blob/8.0.0-beta.1/MIGRATION.md#1-version-support-changes).
+
+### Important Changes
+
+- **feat(svelte): Add Svelte 5 support (#11807)**:
+
+We now officially support Svelte 5.
+
+- **feat(node): Support Node 22 (#11871)**:
+
+We now officially support Node 22.
+
+- **fix(node): Fix nest.js error handler (#11874)**
+
+Errors in Nest.js were sent to Sentry, but the application itself was not working as expected anymore. Now you'll have
+to pass the Nest.js `BaseExceptionFilter` like this:
+
+```typescript
+import { BaseExceptionFilter, HttpAdapterHost } from '@nestjs/core';
+
+const { httpAdapter } = app.get(HttpAdapterHost);
+Sentry.setupNestErrorHandler(app, new BaseExceptionFilter(httpAdapter));
+```
+
+- **feat(browser): Send standalone fetch and XHR spans if there's no active parent span (#11783)**
+
+Starting with this version, spans for outgoing fetch/xhr requests will be captured even if no pageload/navigation span
+is ongoing. This means that you will be able to have a more complete trace, especially for web applications that make a
+lot of HTTP requests on longer-lived pages.
+
+- **feat(browser): Add INP support for v8 (#11650)**
+
+INP web vital support was now forward-ported to version 8. Recording of INP data is enabled by default.
+
+- **feat(core): Increase default transport buffer size from 30 to 64 (#11764)**
+
+The default limit of queued events to be sent was increased from 30 to 64 events. You may observe a higher memory
+footprint of the SDK. You can override this limit by setting the `transportOptions.bufferSize` option in
+`Sentry.init()`.
+
+- **feat(replay): Add "maxCanvasSize" option for replay canvases (#11617)**:
+
+A `maxCanvasSize` option was added to the `replayCanvasIntegration` to disallow capturing of canvases larger than a
+certain size. This value defaults to `1280` which will not capture canvases bigger than 1280x1280 pixels.
+
+- **feat(opentelemetry): Add `addOpenTelemetryInstrumentation` (#11667)**
+
+A utility function `addOpenTelemetryInstrumentation` was added that allows for the registration of instrumentations that
+conform to the OpenTelemetry JS API without having to specify `@opentelemetry/instrumentation` as a dependency.
+
+- **ref(core): Don't start transaction for trpc middleware (#11697)**
+
+Going forward, the Sentry `trpcMiddleware` will only create spans. Previously it used to always create a transaction.
+This change was made to integrate more nicely with the HTTP instrumentation added in earlier versions to avoid creating
+unnecessary transactions.
+
+- **feat(browser): Update `propagationContext` on `spanEnd` to keep trace consistent**
```

Comment:
really all of these should probably be under "other changes" I believe, coming here you have no context of all the other hundreds of changes in the rc 😅